eidetic-daniel-7:insight_testsuite danielchild$ bash ./run_tests.sh

CURRENT WORKING: /Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp
CURRENT DIRECTORY: src

REL BATCH FILE: src/../log_input/batch_log.json
REL STREAM FILE: src/../log_input/stream_log.json
REL FLAGGED FILE: src/../log_output/flagged_purchases.json

ABS BATCH: /Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_input/batch_log.json
ABS STREAM: /Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_input/stream_log.json
ABS FLAGGED: /Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_output/flagged_purchases.json

Initiating handler.
Processing batch data...

Processing stream data...
During stream processing relative stream file location is
	src/../log_input/stream_log.json

and the absolute stream file location is
	/Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_input/stream_log.json

Within handle data, when about to write, destination is
	src/../log_output/flagged_purchases.json

and the absolute write destination is 
	/Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_output/flagged_purchases.json


During writeData relative write destination is:
	src/../log_output/flagged_purchases.json

and the absolute write destination is
	/Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_output/flagged_purchases.json

data being written is:
 {"event_type": "purchase", "timestamp": "2017-06-13 11:33:02", "id": "2", "amount": "1601.83", "mean": "29.1", "sd": "21.46"}

Time elapsed to process batch and stream 0.00333
[PASS]: test_1

CURRENT WORKING: /Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp
CURRENT DIRECTORY: src

REL BATCH FILE: src/../log_input/batch_log.json
REL STREAM FILE: src/../log_input/stream_log.json
REL FLAGGED FILE: src/../log_output/flagged_purchases.json

ABS BATCH: /Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_input/batch_log.json
ABS STREAM: /Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_input/stream_log.json
ABS FLAGGED: /Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_output/flagged_purchases.json

Initiating handler.
Processing batch data...

Processing stream data...
During stream processing relative stream file location is
	src/../log_input/stream_log.json

and the absolute stream file location is
	/Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_input/stream_log.json

Within handle data, when about to write, destination is
	src/../log_output/flagged_purchases.json

and the absolute write destination is 
	/Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_output/flagged_purchases.json


During writeData relative write destination is:
	src/../log_output/flagged_purchases.json

and the absolute write destination is
	/Users/danielchild/Desktop/INSIGHT PROBLEM/anomaly_detection/insight_testsuite/temp/log_output/flagged_purchases.json

data being written is:
 {"event_type": "purchase", "timestamp": "2017-06-13 11:33:02", "id": "2", "amount": "1601.83", "mean": "29.1", "sd": "21.46"}

Time elapsed to process batch and stream 0.00142
[FAIL]: your-own-test
--- ./temp/log_output/flagged_purchases.json	2017-07-08 15:13:02.000000000 -0400
+++ ./tests/your-own-test/log_output/flagged_purchases.json	2017-07-08 15:12:52.000000000 -0400
@@ -1 +0,0 @@
-{"event_type": "purchase", "timestamp": "2017-06-13 11:33:02", "id": "2", "amount": "1601.83", "mean": "29.1", "sd": "21.46"}
[Sat Jul  8 15:13:02 EDT 2017] 1 of 2 tests passed